app/src/layouts/organizationsControl/organizationsPopover/organizationsPopover.jsx

```diff
@@ -45,15 +48,59 @@ export const OrganizationsPopover = ({ closePopover, closeSidebar }) => {
   const availableProjects = useSelector(availableProjectsSelector);
   const currentOrganization = useSelector(urlOrganizationSlugSelector);
   const projectSlug = useSelector(urlProjectSlugSelector);
+  const [valueSearch, setValueSearch] = useState('');
   const maxHeightPopover = window.innerHeight - MARGIN_TOP_AND_MARGIN_BOTTOM;
 
+  const filteredProjects = useMemo(
+    () =>
+      availableProjects.reduce((acc, { organizationSlug, organizationName, projects }) => {
+        const isOrganization = organizationName.toLowerCase().includes(valueSearch.toLowerCase());
+        const searchProjects = projects.filter(({ projectName }) =>
+          projectName.toLowerCase().includes(valueSearch.toLowerCase()),
+        );
+
+        return isOrganization || searchProjects.length > 0
+          ? [
+              ...acc,
+              {
+                organizationSlug,
+                organizationName,
+                projects: searchProjects,
+              },
+            ]
+          : acc;
+      }, []),
+    [valueSearch, availableProjects],
+  );
+
   const onClose = () => {
     closeSidebar();
     closePopover();
   };
 
+  const handleChange = (event) => {
+    setValueSearch(event.target.value);
+  };
+
+  const handleClear = () => {
+    setValueSearch('');
+  };
+
   return (
     <div className={cx('organizations-popover')}>
+      <div className={cx('organizations-search')}>
+        <ThemeProvider theme="dark">
+          <FieldText
+            startIcon=<SearchIcon />
```

Comment:
To be consistent with other places I suggest wrap this to a curly braces.
```suggestion
            startIcon={<SearchIcon />}
```